examples/jsm/nodes/materials/FatPointsNodeMaterial.js

```diff
@@ -0,0 +1,163 @@
+import NodeMaterial, { addNodeMaterial } from './NodeMaterial.js';
+import { varying } from '../core/VaryingNode.js';
+import { property } from '../core/PropertyNode.js';
+import { attribute } from '../core/AttributeNode.js';
+import { cameraProjectionMatrix } from '../accessors/CameraNode.js';
+import { materialColor } from '../accessors/MaterialNode.js';
+import { modelViewMatrix } from '../accessors/ModelNode.js';
+import { positionGeometry } from '../accessors/PositionNode.js';
+import { smoothstep } from '../math/MathNode.js';
+import { tslFn, vec2, vec4 } from '../shadernode/ShaderNode.js';
+import { uv } from '../accessors/UVNode.js';
+import { materialPointWidth } from '../accessors/FatPointsMaterialNode.js'; // or should this be a property, instead?
+import { viewport } from '../display/ViewportNode.js';
+
+import { PointsMaterial } from 'three';
+
+const defaultValues = new PointsMaterial();
+
+class FatPointsNodeMaterial extends NodeMaterial {
```

Comment:
Could this be extended of `PointsNodeMaterial`?

Reply:
Maybe. I think I need you to specify the design you prefer.

Be advised, I expect "instanced sprites" will be implemented at some point. For now, I am only focusing on "fat points", sized in pixel units.

Reply:
I get it, for now I think it would be better if it was extended.

Perhaps we could move these new classes of `/points/` separately by categories?

```js
jsm/objects/FatPoints.js
jsm/geometries/FatPointsGeometry.js
```

Reply:
I'm thinking that if this PR is generally acceptable, I should remove the Draft designation, wait for your approval, merge it, and then let you refactor.

How does that plan sound?

Reply:
> How does that plan sound?

Sure, sounds good to me.

Reply:
OK, resolving...